virttest/qemu_devices/qcontainer.py

```diff
@@ -2735,6 +2754,10 @@ def pcic_by_params(self, name, params, parent_bus=None):
                                 parent_bus=parent_bus,
                                 child_bus=bus)
 
+    def throttle_group_define_by_params(self, group_params, name):
+        group_param = group_params.get("throttle_group_parameter")
+        return QThrottleGroup(name, group_param if group_param else "{}")
```

Comment:
```suggestion
        group_param = json.loads(group_params.get("throttle_group_parameter", "{}"))
        return QThrottleGroup(name, group_param)
```

Reply:
group_params.get("throttle_group_parameter", "{}") can not handle empty string like throttle_group_parameter_group4 =

Reply:
updated